spec/index.bs

```diff
@@ -3286,11 +3319,19 @@ A {{GPUProgrammableStageDescriptor}} describes the entry point in the user-provi
             <dl class=switch>
                   : {{GPUBindingType/"sampler"}}
                   :: the |binding| is a non-comparison sampler
+                  : {{GPUBindingType/"filtering-sampler"}}
+                  :: the |binding| is a non-comparison sampler, and it's only used, statically, for
+                      sampling textures bound as {{GPUBindingType/"filtered-texture"}}.
                   : {{GPUBindingType/"comparison-sampler"}}
                   :: the |binding| is a comparison sampler
                   : {{GPUBindingType/"sampled-texture"}}
-                  :: the |binding| is a sampled texture with the component type of |entry|.{{GPUBindGroupLayoutEntry/textureComponentType}}
+                  :: The |binding| is a sampled texture with the component type of |entry|.{{GPUBindGroupLayoutEntry/textureComponentType}},
                       and it has to have a sample count of 1.
```

Comment:
hm, what sample count does this refer to? This is talking about the binding type inside the shader, right? So rather than sample count I think it should say it's non-multisampled (or more precisely refer to the actual type from WGSL).

Reply:
indeed! Not directly related to the PR though